src/test/java/endpoints/UserAuthEndpoints.java

```diff
@@ -1,38 +1,52 @@
 package endpoints;
 
-import io.restassured.http.ContentType;
 import io.restassured.response.Response;
+import org.openqa.selenium.remote.http.HttpMethod;
 import payload.UserCredentials;
 
-import static io.restassured.RestAssured.given;
-
 public class UserAuthEndpoints {
-    public static Response getUserToken(UserCredentials userCredentials){
-        return given()
-                .contentType(ContentType.JSON)
+
+    public static Response getUserToken(UserCredentials userCredentials) {
+
+        AysRestAssuredRequest restAssuredRequest = AysRestAssuredRequest.builder()
+                .httpMethod(HttpMethod.POST)
+                .url(Routes.authUserToken)
                 .body(userCredentials)
-                .when()
-                .post(Routes.authUserToken);
+                .build();
+
+        return AysRestAssured.perform(restAssuredRequest);
+
     }
-    public static Response userTokenRefresh(String refreshToken){
-        return given()
-                .contentType(ContentType.JSON)
-                .body("{\n" +
-                        "    \"refreshToken\": \""+refreshToken+"\"\n" +
-                        "}")
-                .when()
-                .post(Routes.authUserTokenRefresh);
 
+    public static Response userTokenRefresh(String refreshToken) {
+
+        AysRestAssuredRequest restAssuredRequest = AysRestAssuredRequest.builder()
+                .httpMethod(HttpMethod.POST)
+                .url(Routes.authUserTokenRefresh)
+                .body(new RefreshTokenRequest(refreshToken))
+                .build();
+
+        return AysRestAssured.perform(restAssuredRequest);
     }
-    public static Response userInvalidateToken(String accessToken,String refreshToken){
-        return given()
-                .header("Authorization", "Bearer " + accessToken)
-                .contentType(ContentType.JSON)
+
+    public static Response userInvalidateToken(String accessToken, String refreshToken) {
+
+        AysRestAssuredRequest restAssuredRequest = AysRestAssuredRequest.builder()
+                .httpMethod(HttpMethod.POST)
+                .url(Routes.authUserTokenInvalidate)
                 .body("{\n" +
-                        "    \"refreshToken\": \""+refreshToken+"\"\n" +
+                        "    \"refreshToken\": \"" + refreshToken + "\"\n" +
                         "}")
```

Comment:
@Nbhtozcn @cagla88 Burada JSON hazırlamak yerine obje kullanmalıyız, bir tane örneğini yapmıştım. Orayı inceleyin isterseniz, onun üzerine konuşalım tekrar.

Reply:
Dediğin ornegi hatırlayamadım Agit.Biz projede request bodyler icin POJO classlar kullanıyoruz, burda da onu uyguladim ve obje göndermiş olacağız, tekrar review yaparsın uygun degilse değiştiririz.